.circleci/config.yml

```diff
@@ -56,6 +56,52 @@ jobs:
             git config --local user.email "sysadmin@binary.com"
             git config --local user.name "gh-pages deploy bot"
             make gh-pages
+  update_schema:
+    <<: *default
+    steps:
+      - checkout
+      - run:
+          name: config git
+          command: |
+            git config --global user.email "nobody@deriv.com"
+            git config --global user.name "Nobody"
+      - run:
+          name: update schema
+          command: |
+            git clone https://github.com/binary-com/deriv-developers-portal.git /tmp/deriv-developers-portal
+            curl -L https://cpanmin.us | perl - --sudo App::cpanminus
+            sudo cpanm -n Dir::Self File::Basename JSON::MaybeXS Log::Any Path::Tiny Template Syntax::Keyword::Try
```

Comment:
Does it make sense to have a cpanfile for installing these dependencies ? Or is the dependencies too little to have a separate file for it ?

Reply:
This is a python project. I'm installing dependencies for that script `scripts/regen-py.pl` . I don't think its necessary to maintain a cpanfile.